### ISSUE TITLE
Zip call in archive.py not transferring src or examples folder

### ISSUE THREAD

**Issue by Craig44** [NIWAFisheriesModelling/CASAL2#132]

**Comment by Craig44** [comment]
When the archive build runs everything works fine up until line: 127 in Archiver.py
This command does not store Examples or Source code in the zipped file.

**Comment by Zaita** [comment]
Firstly, this works as expected on my machine. Please provide more context/print out of the error.

Secondly, we've discussed this numerous times and the decision has always been to not include the source code in the archive. A break down of the argument has always been this

Including source code
- Someone can read the code
- All source control is stripped
- The code cannot be compiled
- The code will not work in an IDE with code highlighting/indexing etc
- Anyone with the code will have to submit changes via email to NIWA without compiling or testing
- Linking the source code to GIT requires quite a bit of familiarity with GIT
- Not possible to use Subversion
- Cannot rebuild CASAL2 manual with any changes they make

Not including the source code
- Anyone can read the source code via the GitHub URL (no login or account required). Has nice layout and source highlighting
- All source control is still maintained on all copies of the source code controlled by NIWA/MPI
- The code can be compiled
- The code will work an in an IDE with code highlighting/indexing etc
- Anyone with a GitHub account can make a pull request to NIWA with their code changes after they've compiled and tested them.
- Someone can run casal -v to get the version of CASAL2 and lookup that version of the source code online
- If the user wants a zipped copy of the source code, they can download it via GitHub with full source control intact and all dependencies for compiling
- Can use SubVersion
- Can rebuild CASAL2 manual with code changes they make

**Comment by Craig44** [comment]
![zip_message](https://cloud.githubusercontent.com/assets/13144077/15873175/6f05a3fe-2d51-11e6-99d7-3774cdc7d2f9.png)
![empty_examples_folder](https://cloud.githubusercontent.com/assets/13144077/15873188/7d43034e-2d51-11e6-9888-328e7f0d1ab5.png)

That's fine about the source code. Attached are the messages I get when running that zip command.I believe the jenkins does the same, because I stumbled across this issue when the archived casal2.zip had an empty example folder. I have turned off the archive for windows on the jenkins system and am manually ftp ing the most recent build manually till this is fixed.
